sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_connection.py

```diff
@@ -188,13 +188,13 @@ def _connect(self):
             else:
                 self._set_state(ConnectionState.HDR_SENT)
         except (OSError, IOError, SSLError, socket.error) as exc:
+            if isinstance(exc, FileNotFoundError) and exc.filename and "ca_certs" in exc.filename:
```

Comment:
added for parity with uamqp. Checking specifically for "ca_certs", b/c we're not sure if other cert/key files would raise the same error but be handled differently + not sure how to test that.

Reply:
Could you add that as a comment into the code?

Reply:
Essentially any time you want to add a comment to a PR to explain something - that should probably be documented in the code itself.